plugins/TimingTriggerCandidateMaker.cpp

```diff
@@ -33,17 +33,23 @@ TimingTriggerCandidateMaker::HSIEventToTriggerCandidate(const dfmessages::HSIEve
 {
   triggeralgs::TriggerCandidate candidate;
   // TODO Trigger Team <dune-daq@github.com> Nov-18-2021: the signal field ia now a signal bit map, rather than unique value -> change logic of below?
-  if (m_detid_offsets_map.count(data.signal_map)) {
-    // clang-format off
-    candidate.time_start = data.timestamp - m_detid_offsets_map[data.signal_map].first;  // time_start
-    candidate.time_end   = data.timestamp + m_detid_offsets_map[data.signal_map].second; // time_end,
-    // clang-format on
+  if (m_hsi_passthrough == true){
+    TLOG_DEBUG(3) << "HSI passthrough applied, modified readout window is set";
+    candidate.time_start = data.timestamp - hsi_pt_before;
+    candidate.time_end   = data.timestamp + hsi_pt_after;
   } else {
-    throw dunedaq::trigger::SignalTypeError(ERS_HERE, get_name(), data.signal_map);
+    if (m_detid_offsets_map.count(data.signal_map)) {
+      // clang-format off
+      candidate.time_start = data.timestamp - m_detid_offsets_map[data.signal_map].first;  // time_start
+      candidate.time_end   = data.timestamp + m_detid_offsets_map[data.signal_map].second; // time_end,
+      // clang-format on    
+    } else {
+      throw dunedaq::trigger::SignalTypeError(ERS_HERE, get_name(), data.signal_map);
```

Comment:
I think this can be `ers::error(dunedaq::trigger::SignalTypeError(ERS_HERE, get_name(), data.signal_map));` instead: that will report an error to the logging system without aborting the thread.

Reply:
I've added this.
Keep in mind there is a catch logic associated with this:
catch (SignalTypeError& e) {
    m_tc_sig_type_err_count++;
    ers::error(e);
    return;
which now won't get accessed? so the counter here won't count properly.

Reply:
If you'd like we can move the counter incrementation right after the ers::error( SignalTypeError ) call  and remove the catch completely, let me know.

Reply:
Oh, my mistake. You were right the first time. I'll put it back how it was